docs/index.md

```diff
@@ -1578,17 +1580,17 @@ mocha.setup({
 
 ### Browser-specific Option(s)
 
-Browser Mocha supports many, but not all [cli options](#command-line-usage).  
+Browser Mocha supports many, but not all [cli options](#command-line-usage).
 To use a [cli option](#command-line-usage) that contains a "-", please convert the option to camel-case, (eg. `check-leaks` to `checkLeaks`).
 
 #### Options that differ slightly from [cli options](#command-line-usage):
 
-`reporter` _{string|constructor}_  
```

Comment:
Could you revert these? Removes the new-line so its harder to read.

Reply:
Done.  Sorry about that.  My editor removes trailing whitespace automatically.  'Didn't realize it was significant in Markdown.